src/views/templates/utils/index.ts

```diff
@@ -18,10 +17,17 @@ export const isDedicatedCPUPlacement = (template: V1Template): boolean =>
 // check if the Descheduler is installed
 export const useDeschedulerInstalled = (): boolean => {
   const [resourceList] = useK8sWatchResource<any>({
-    kind: KubeDeschedulerModel.kind,
+    kind: 'Namespace',
```

Comment:
I am changing it because it returned an empty array, no matter the operator is installed or not => this function does not work properly - also mentioned in the BZ by Guohua. Also after the discussion with Ruth some time ago, I was told to check `openshift-kube-descheduler-operator` namespace if I want to check if the operator is installed. Not sure, why it was not implemented like that. Probably I was suggested to implement it this way (and not any other way) by Kobi or someone else.

I'll try to better use `groupVersionKind` as suggested. Thank you! :)

Reply:
Now I remember that you've suggested me to better not search for the namespace, as there is a possibility that someone would create this namespace and not install the operator, or uninstall the operator without deleting the namespace, so maybe we need to find some other solution. WDYT?

Reply:
I think we have a bug where the `useK8sWatchResource` hook is not refreshing the request when isList is set to true, not sure what is the root cause for that.
anyway I tried locally this block of code and it solves the problem:
```
export const useDeschedulerInstalled = (): boolean => {
  const [resourceList] = useK8sWatchResource<any>({
    groupVersionKind: KubeDeschedulerModelGroupVersionKind,
    isList: false,
  });
  
  return !isEmpty(resourceList);
};
```

also, I would recommend renaming this file, index files usually to have a single file for exports so we can have shorter import lines, I would have changed it to utils.ts
also please extract this hook to another file hooks and functions should not be in the same file